examples/knative/go/kn-go-echo/README.md

```diff
@@ -1,83 +1,121 @@
 # kn-go-echo
+
 Simple Go function running in Knative to echo [CloudEvents](https://github.com/cloudevents/sdk-go).
-In addition, [Buildpacks](https://buildpacks.io) are used to create the artifacts.
+In addition, [ko](https://github.com/google/ko) is used to create the artifacts.
 
 Valid events will be printed out JSON-encoded, separated by their CloudEvent
 `attributes` and `data` for better readability.
+
 ## Deployment (Knative)
 
 **Note:** The following steps assume a working Knative environment using the
 `default` broker. The Knative `service` and `trigger` will be installed in the
 `default` Kubernetes namespace, assuming that the broker is also available there.
 
 ```bash
-# create the service
+# Create the service.
 kn service create kn-go-echo --port 8080 --image vmware/veba-kn-go-echo:latest
 
-# create the trigger
+# Create the trigger.
 kn trigger create kn-go-echo --broker default --sink ksvc:kn-go-echo
 ```
 
-## Build with `pack`
+## Build with `ko`
+
+`ko` was created to help Knative developers build images and binaries without
+the need for Dockerfiles nor Makefiles. See more information in [this Knative
+blog](https://knative.dev/blog/2018/12/18/ko-fast-kubernetes-microservice-development-in-go/).
+
+Follow setup instructions at https://github.com/google/ko).
 
-- Requirements:
-  - `pack` (see: https://buildpacks.io/docs/app-developer-guide/)
-  - Docker
+After installing ko and setting the appropriate environmental variables for it,
+run the following command from the kn-go-echo directory. It will build and push
+the image to your image registry.
 
 ```bash
-IMAGE=vmware/veba-kn-go-echo:latest
-pack build -B gcr.io/buildpacks/builder:v1 ${IMAGE}
+ko publish ../kn-go-echo
 ```
 
-## Verify the image works by executing it locally
+To run the container, use
 
 ```bash
-docker run -e PORT=8080 -it --rm -p 8080:8080 vmware/veba-kn-go-echo:latest
+docker run -p 8080:8080 $(ko publish ../kn-go-echo)
+```
+
+In a separate window, send a POST request.
 
-# now in a separate window or use -d in the docker cmd above to detach
+```bash
 $ curl -X POST -i localhost:8080
-HTTP/1.1 204 No Content
-Date: Fri, 02 Apr 2021 21:15:20 GMT
+HTTP/1.1 415 Unsupported Media Type
+Date: Thu, 06 May 2021 21:12:22 GMT
+Content-Length: 0
+```
+
+You can also send a fake cloud event.
```

Comment:
Please only show this example as the other one errors out which could be misleading

Reply:
Sounds good